### DISCUSSION TITLE
How do I use ContinuousSpace in my model?

### DISCUSSION THREAD

**Discussion by ghost** [mesa/mesa#1680] [Q&A] [answered]
My code looks like this:

<img width="879" alt="image" src="https://user-images.githubusercontent.com/131771458/235376127-9cd830a1-4515-4e9c-a593-d34ae4fed281.png">
How do I choose an appropriate canvas grid for a ContinuousSpace-type grid? I am unable to locate the correct option from the options shown below:

<img width="462" alt="image" src="https://user-images.githubusercontent.com/131771458/235376186-f496b315-a93d-4e46-8be4-3b789744c89a.png">

**Comment by rht** [comment, marked as answer]
You can check out the [boid_flockers](https://github.com/projectmesa/mesa-examples/tree/main/examples/boid_flockers) example for `ContinuousSpace`.

**Reply by ghost** [reply]
Thank you so much!
